README.md

```diff
@@ -1,2 +1,23 @@
-# discordjs-bot-bootstrap
-A bootstrap for DiscordJS bots
+# Spike 3
+A Discord bot built prodominatly by students, former students, and alumni of Truman State University
+
+## What is Spike?
+Spike is a Discord bot used to automate server management, boost member activity, and increase member retention.
+
+## Installation
+First install [NodeJs](https://www.npmjs.com/package/discord.js/v/12.5.3)
```

Comment:
This link should point to the nodejs main page, not the discord.js package page.